tests/fixtures/ember-container-query-glint/input/src/components/products/product/image.hbs

```diff
@@ -1,12 +1,13 @@
 {{#if this.isTestEnvironment}}
-  <div local-class="placeholder-image"></div>
+  <div local-class="placeholder-image" ...attributes></div>
```

Comment:
This component covers an edge case of `Element`. There is dynamic code as well as more than 1 `...attributes`, so it's hard to guarantee correctness.

https://github.com/emberjs/rfcs/blob/master/text/0748-glimmer-component-signature.md#components-with-multiple-or-varying-elements